test/buildah-bud/apply-podman-deltas

```diff
@@ -293,6 +293,10 @@ skip_if_remote "FIXME #25138: mount cache not working one remote" \
 ###############################################################################
 # BEGIN temporary workarounds that must be reevaluated periodically
 
+#2025-04-21 Broken in main, see https://github.com/containers/podman/pull/25912
+skip "FIXME: This test should be fixed in Podman" \
+	"bud and test inherit-labels"
```

Comment:
This needs to be removed, if the options is wired in that test must pass.  With this we do not know that your change works.

Reply:
Fingers crossed, removed.